app/tour_step_directive.js

```diff
@@ -91,7 +91,8 @@
 
                     //Add step to tour
                     templateReady.then(function () {
-                        ctrl.addStep(step);
+						ctrl.addStep(step);
+						scope.step = step;
```

Comment:
Can you change this to "scope.tourStep"? I think "step" will be too general and might override other scope properties. It looks like you are using tabs instead of spaces, can you replace with 4 spaces, please? I will merge after these changes are made.